From 82fc884246176e96e3ab77c5aaf8d619049e85cc Mon Sep 17 00:00:00 2001
From: Ilkka Koskinen <ilkka.koskinen@intel.com>
Date: Thu, 18 Sep 2014 18:32:56 -0700
Subject: [PATCH 097/117] atomisp2: Fix pm_qos handling in probe()'s failure
 path

The driver tried to remove pm_qos settings twice

Change-Id: I2fd33e7b8ae62045ce311de1a3ac8b1f50c536a1
Signed-off-by: Ilkka Koskinen <ilkka.koskinen@intel.com>
---
 .../camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_v4l2.c    | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_v4l2.c b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_v4l2.c
index ea0eeb2..57049d6 100644
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_v4l2.c
+++ b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_v4l2.c
@@ -1431,9 +1431,6 @@ wdt_work_queue_fail:
 fw_validation_fail:
 	release_firmware(isp->firmware);
 load_fw_fail:
-#ifdef CONFIG_GMIN_INTEL_MID
-	pm_qos_remove_request(&isp->pm_qos);
-#endif
 	pci_dev_put(isp->pci_root);
 	return err;
 }
-- 
1.9.3

